llvm/test/CodeGen/AArch64/GlobalISel/endian_fallback.ll

```diff
@@ -0,0 +1,18 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple aarch64-unknown-linux-musl -O0 < %s | FileCheck %s --check-prefix=CHECK-LE
+; RUN: llc -mtriple aarch64_be-unknown-linux-musl -O0 < %s | FileCheck %s --check-prefix=CHECK-BE
+
+; Make sure we fall-back to SDAG for BE targets.
```

Comment:
This doesn't really do that unless you check for the fallback remark

Reply:
Sorry - I missed your remark earlier, my bad. It was crashing earlier so was fairly obvious that it was "fixed". Let me add a check for the remark too.